packages/app/src/messages/components/messages/messages.component.tsx

```diff
@@ -525,15 +525,15 @@ const Messages: FunctionComponent<MessagesProps> = ({
   }
 
   const getViewReceiver = (activeThread: Thread): Receiver => {
+    const receiver = getReceiver(activeThread.phoneNumber)
+
     if (activeThread.id === mockThread.id) {
       return {
-        phoneNumber: activeThread.phoneNumber,
+        ...(receiver || { phoneNumber: activeThread.phoneNumber }),
```

Comment:
wonder if you have to do object destructuring, maybe simply do this:

`phoneNumber: received.phoneNumber ?? activeThread.phoneNumber`

Reply:
unfortunately, it won't work. the goal of this line is to provide additional fields if the receiver has been found. in other cases will provide only phoneNumber